### DISCUSSION TITLE
How can I access the older documentation of Gluestack ui ?

### DISCUSSION THREAD

**Discussion by Ali-Aref** [gluestack/gluestack-ui#1999] [Q&A] [answered]
I were using native-base ui and migrated to gluestack ui in the early development stages. I re-wrote all my nativebase codes for gluestack ui. Now there are amazing updates in gluestack but, my project is getting bigger and I don't have time to re-write it again. 

How can access the docs where "." notion were used. as 
```
<Button>
  <Button.Text mx="$2" color="$black">
    Save
  </Button.Text>
</Button>
```

is that docs still available ?

**Comment by vaibhk20** [comment, marked as answer]
👋 Hi @Ali-Aref ! We've noticed you're using gluestack-ui v1. You can find the documentation for v1 [here](https://v1.gluestack.io/ui/docs/home/overview/introduction)

We've recently released gluestack-ui v2! The new version adds React Server Components (RSC) compatibility for improved performance and SEO, along with easier styling using NativeWind that follows Tailwind CSS conventions. We recommend checking it out. You can find the migration guide for moving from v1 to v2 [here](https://gluestack.io/ui/docs/home/overview/upgrade-to-v2).
